citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -532,7 +536,9 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
       			out.println("\t\t\t</table>");
        		    out.println("\t\t</div>");
 
-			    out.println("\t\t<div class=\"availabilityHeader\"></div>");
+
+			    out.println("\t\t<div class=\"availabilityHeader\" style=\"color:" +
+					    ServerConfigurationService.getString("official.institution.background.colour") + "\">" + "<strong>Availability</strong>" + "</div>");
```

Comment:
Is this a new property you've created?
If so it probably needs a better name as it just affects the citations tool so it should probably have a "citations." prefix (or whatever the citations prefix is for other config options).